### ISSUE TITLE
3.6 broke keyword functionality.

### ISSUE THREAD

**Issue by pstaubs** [Uberi/speech_recognition#198]
3.6 broke keyword functionality.

Please run the sample script on the official document page https://github.com/Uberi/speech_recognition/blob/master/examples/special_recognizer_features.py, e.g.:

```
#!/usr/bin/env python3

import speech_recognition as sr

from os import path
AUDIO_FILE_EN = path.join(path.dirname(path.realpath(__file__)), "english.wav")

# use the audio file as the audio source
r = sr.Recognizer()
with sr.AudioFile(AUDIO_FILE_EN) as source:
    audio_en = r.record(source) # read the entire audio file

# recognize keywords using Sphinx
try:
    print("Sphinx recognition for \"one two three\" with different sets of keywords:")
    print(r.recognize_sphinx(audio_en, keyword_entries=[("one", 1.0), ("two", 1.0), ("three", 1.0)]))
    print(r.recognize_sphinx(audio_en, keyword_entries=[("wan", 0.95), ("too", 1.0), ("tree", 1.0)]))
    print(r.recognize_sphinx(audio_en, keyword_entries=[("un", 0.95), ("to", 1.0), ("tee", 1.0)]))
except sr.UnknownValueError:
    print("Sphinx could not understand audio")
except sr.RequestError as e:
    print("Sphinx error; {0}".format(e))
```

Error:

```
Traceback (most recent call last):
  File "E:/Documents/workspace/Alphred/python/test.py", line 16, in <module>
    print(r.recognize_sphinx(audio_en, keyword_entries=[("one", 1.0), ("two", 1.0), ("three", 1.0)]))
  File "c:\program files\python27\lib\site-packages\speech_recognition\__init__.py", line 671, in recognize_sphinx
    decoder.set_kws("keywords", f.name)
  File "c:\program files\python27\lib\site-packages\pocketsphinx\pocketsphinx.py", line 404, in set_kws
    return _pocketsphinx.Decoder_set_kws(self, name, keyfile)
RuntimeError: Decoder_set_kws returned -1
```

I tried this on two machines now after updating to 3.6. So far my solution is to downgrade to 3.5.

**Comment by ZachDaChampion** [comment]
I'm having the same problem. I'll try downgrading, thanks. Never tried this before 3.6 so I thought I was doing something wrong

**Comment by Uberi** [comment]
This should be fixed as of version 3.6.5!
